lib/matplotlib/backend_bases.py

```diff
@@ -579,9 +579,10 @@ def _get_text_path_transform(self, x, y, s, prop, angle, ismath):
         path = Path(verts, codes)
         angle = np.deg2rad(angle)
         if self.flipy():
+            width, height = self.get_canvas_width_height()
             transform = Affine2D().scale(fontsize / text2path.FONT_SCALE,
                                          fontsize / text2path.FONT_SCALE)
-            transform = transform.rotate(angle).translate(x, self.height - y)
```

Comment:
A few tests broke.  Are you sure this change is correct?  Its the only thing I can see that might have broken them...

Reply:
failures seem spurious now

Reply:
Restarted the failing test. The commit looks good overall, but I do not fully understand the change in the height-handling. Thus, at least I want to see that all tests pass.